diff --git a/root/etc/uci-defaults/luci-app-vlmcsd b/root/etc/uci-defaults/luci-app-vlmcsd
index d28b2ec..df07faf 100755
--- a/root/etc/uci-defaults/luci-app-vlmcsd
+++ b/root/etc/uci-defaults/luci-app-vlmcsd
@@ -10,7 +10,7 @@ EOF
 uci delete firewall.kms
 uci add firewall rule 
 uci rename firewall.@rule[-1]="kms"
-uci set firewall.@rule[-1].name="kms"
+uci set firewall.@rule[-1].name="Allow-KMS"
 uci set firewall.@rule[-1].target="ACCEPT"
 uci set firewall.@rule[-1].src="wan"
 uci set firewall.@rule[-1].proto="tcp"
